tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -177,7 +212,7 @@ spec:
         - image
         - "--verbose"
         - "--images"
-        - "$(params.IMAGES)"
+        - "$(params.HOMEDIR)/snapshot.json"
```

Comment:
Should this be `$(params.SNAPSHOT_PATH)`?

Also, not sure if we want to expose this as a parameter to the user. It seems more like an implementation detail.

Reply:
for `Integration Test Scenarios`, SNAPSHOT is the json representation of the snapshot.
for `Release Pipelines`, SNAPSHOT will be the path to the json file.

With my latest PR #1972 , I am always outputting a JSON file that will either contain the **complete** `Snapshot` or the **reduced** one.

so yes, once that PR merges, I can use `$(params.SNAPSHOT_PATH)`.

> It seems more like an implementation detail

Are you meaning to say it should not a param, and I am simply hard-code `/tekton/home/snapshot.json`?

Reply:
> Are you meaning to say it should not a param, and I am simply hard-code /tekton/home/snapshot.json?

Yeah, exactly.

Reply:
removed parameter and using the already defined `HOME` parameter in the arg